.gitignore

```diff
@@ -154,4 +154,6 @@ bld/
 # OpenCover
 *_coverage.xml
 
-tsdoc-metadata.json
\ No newline at end of file
+tsdoc-metadata.json
+
+packages/E2ETest/reports/
```

Comment:
If you added the line in `packages/E2ETest/.gitignore`, do you need to also add it here?

Reply:
I don't know why it doesn't work in my computer, and that's why you see it on both place.
I will try to remove one of them in next PR.